html/css/product-multilingual.css

```diff
@@ -39,3 +39,125 @@ div.tagsinput input:focus {
     background: none;
     border: none;
 }
+
+.ui-selectable { list-style-type: none; margin: 0; padding: 0; }
+
+.ui-selectable img {
+  vertical-align:middle;
+}
+.ui-selectee:hover { background: #FECA40; }
+.ui-selected { background: #F39814; color: white; }
+
+.command { margin-bottom:5px; }
+
+label { margin-top: 20px; }
+
+fieldset { margin-top: 15px; margin-bottom:15px;}
+
+legend { font-size: 1.375em; margin-top:2rem; }
+
+textarea {  height:8rem; }
+
+.display { float:right; margin-top:10px;margin-bottom:10px; max-width:400px; }
+.cropbox { max-width:100%; }
+.ingredients_image_full { max-height:400px }
+
+.upload_image_div {
+	padding-top:0.5rem;
+}
+
+.button_div {
+	margin-top:0.5rem;
+}
+
+#label_new_code, #new_code { display: inline; margin-top: 0px; width:200px; }
+
+th {
+	font-weight:bold;
+}
+
+// add borders to Foundation 5 tabs
```

Comment:
I don't think this is a valid CSS comment? https://www.w3.org/TR/CSS21/syndata.html#comments

Reply:
I never know how to make CSS comments. :) I'll fix it.

Reply:
Also is the whole block commented out or the media query only? @stephanegigandet

Reply:
just the media query I think.